docs/pages/testing-playground.vue

```diff
@@ -7,7 +7,17 @@
     Please do not modify the contents of this file.
   -->
   <div id="testing-playground" style="padding: 24px">
-    <component :is="component" v-bind="componentProps" />
+    <component :is="component" v-bind="componentProps">
+      <!-- Render default slot if provided -->
+      <template v-if="slots.default">
+        <!-- eslint-disable-next-line vue/no-v-html -->
+        <span v-html="slots.default"></span>
```

Comment:
Why should this be in as span? I think a div would be more appropiate in case we need to add more elements inside

Reply:
If I take the KButton test with default slot as an example, if you use div, the button will become twice the size, with the upper part showing slot content and the lower part showing the content passed to button itself. That's the reason for using span here.

Reply:
Hmm I see, but there could be some components where a span could also be a problem. What if we add something like a "wrapperEl" prop that defines the element container, and a "content" that defines the v-html prop? Any other idea is welcome :open_hands:.

Reply:
I think we can combine both and have something like:
```javascript
{
  slots: {
    default: {
      element: "div", // or any vue component like KIcon
      elementProps: {
        class: "some-class"
      },
      innerHTML: "<div> Some nested <a>content</a> </div>"
    }
  }
}
```